pkgs/by-name/wa/wavelog/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+  php,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "wavelog";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "wavelog";
+    repo = pname;
+    rev = version;
```

Comment:
rev -> tag

Reply:
I will gladly change this, but can you link to me/explain why this change is better?

Reply:
The motivation is explained in the PR adding that https://github.com/NixOS/nixpkgs/pull/355973

And as you are also having `rev = version`, this change was also covered exactly by the motivation. 

tldr: if you have a branch that has the same name as the version it gets ambiguous what is fetched